docs/roadmap.md

```diff
@@ -1,26 +1,28 @@
 # Roadmap
 
 - [Roadmap](#roadmap)
-  - [v2.3](#v23)
-    - [Merge Argo CD Notifications into Argo CD](#merge-argo-cd-notifications-into-argo-cd)
-    - [Merge ApplicationSet controller into Argo CD](#merge-applicationset-controller-into-argo-cd)
-    - [Compact resources tree](#compact-resources-tree)
-    - [Maintain difference in cluster and git values for specific fields](#maintain-difference-in-cluster-and-git-values-for-specific-fields)
-    - [ARM images and CLI binary](#arm-images-and-cli-binary)
+  - [v2.4](#v24)
     - [Server side apply](#server-side-apply)
-  - [v2.4 and beyond](#v24-and-beyond)
-    - [First class support for ApplicationSet resources](#first-class-support-for-applicationset-resources)
     - [Input Forms UI Refresh](#input-forms-ui-refresh)
-    - [Merge Argo CD Image Updater into Argo CD](#merge-argo-cd-image-updater-into-argo-cd)
     - [Web Shell](#web-shell)
     - [Helm values from external repo](#helm-values-from-external-repo)
-    - [Add support for secrets in Application parameters](#add-support-for-secrets-in-application-parameters)
     - [Config Management Tools Integrations UI/CLI](#config-management-tools-integrations-uicli)
+  - [v2.5 and beyond](#v25-and-beyond)
+    - [First class support for ApplicationSet resources](#first-class-support-for-applicationset-resources)
+    - [Merge Argo CD Image Updater into Argo CD](#merge-argo-cd-image-updater-into-argo-cd)
+    - [Support multiple sources for an Application](#support-multiple-sources-for-an-application)
+    - [Sharding application controller](#sharding-application-controller)
+    - [Add support for secrets in Application parameters](#add-support-for-secrets-in-application-parameters)
     - [Allow specifying parent/child relationships in config](#allow-specifying-parentchild-relationships-in-config)
     - [Dependencies between applications](#dependencies-between-applications)
     - [Multi-tenancy improvements](#multi-tenancy-improvements)
```

Comment:
I don't want to push this one out much further, to be honest.

Reply:
I know it took forever to make progress here. Would you change your mind if we prioritize it and started working on it in e.g. next release? Meaning you won't not have to chase maintainers and wait for feedback on proposal? :)